docker-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
```

Comment:
See if you can make this python version configurable that you can pass it via the shell script or environment variable.

If you look into the docker shell scripts in any of these [examples here](https://github.com/neomatrix369/awesome-ai-ml-dl/tree/master/examples), you can see  how it's done.

Reply:
Thank you for pointing this out. I've updated the Dockerfile to pass the argument via the shell script.